src/BarChart/BarSeries/Bar.tsx

```diff
@@ -576,6 +576,7 @@ export const Bar: FC<Partial<BarProps>> = ({
             onMouseLeave={onMouseLeaveInternal}
             onClick={onMouseClick}
             onMouseMove={onMouseMove}
+            aria-label={JSON.stringify(tooltipData)}
```

Comment:
This is not a viable option

Reply:
Did you test this with a screen reader to see the output?

Reply:
Tested with Mac voice over.

https://github.com/reaviz/reaviz/assets/33908100/dc2e77c7-ac6f-4ada-8c32-1911f7069b31

JSON.stringify didn't look like the best idea to me as well but the idea behind it was to have a uniform format across the labels. We can use the same format we use for tooltips but since that's not uniform and customizable, it wouldn't look the same at all places and may also cause duplicate code as the formatting of the tooltip is done in a different component unlike the rendered graphics.
What do you suggest?

Reply:
@shaswat-indian i think in this case "looking the same" isn't really relevant. What we're going for is customizing the aria-label for each chart so that it's accurate to each chart and is also human friendly. For example, your above example would read out:
`left brace y colon 10 x Phishing Attack right brace graphics symbol`

Besides the reader reading out unnecessary things like braces and brackets, we really just want something like `Phishing Attack: 10`. Now, also imagine if the data, instead of `10` was something like `10.4827492957592927` - we wouldn't want the reader to actually read out each of those decimal places. We'd want it to be similar to how we would format things in a tooltip for non-screen readers too. If you're just JSON.stringify-ing everything then whatever is in the blob will get read.

Reply:
We have a method the Tooltip uses to format the labels that you should be able to leverage directly.